STUDIO/org.openl.rules.webstudio/webapp/pages/modules/copyProject.xhtml

```diff
@@ -253,7 +253,11 @@
                 }
             }
 
-            withCommitInfoRequired('#{studio.currentUsername}', () => copyProjectRepositoryType === "repo-git", afterAction);
+            withCommitInfoRequired(
+                '#{studio.currentUsername}',
+                () => copyProjectRepositoryType === "repo-git" || JSON.parse('#{copyBean.allowedRepositoriesTypes}')[$j('#copyProjectForm\\:repository').val()] === "repo-git",

```

Comment:
Please, pass ``$j('#copyProjectForm\\:repository').val()`` directly to ``copyBean.allowedRepositoriesTypes`` method as parameters and do all required logic there to get rid of redundant JSON serialization and deserialization.
P.S. You will need to define ``copyBean.allowedRepositoriesTypes`` as JS function in JSF

Reply:
Fixed.